TouchID.m

```diff
@@ -20,11 +21,17 @@ @implementation TouchID
 }
 
 RCT_EXPORT_METHOD(authenticate: (NSString *)reason
+                  options:(NSDictionary *)options
                   callback: (RCTResponseSenderBlock)callback)
 {
     LAContext *context = [[LAContext alloc] init];
     NSError *error;
 
+    if ([options objectForKey:@"fallbackLabel"] != nil) {
```

Comment:
@mironiasty This line was dying for me. I modified it to `if (RCTNilIfNull([options objectForKey:@"fallbackLabel"]) != nil) {`  and it started to work. When not passing in an object at all, this JS null needs to get converted into the obj-c nil.

I think it's mergeable after this change!